packages/docusaurus-theme-classic/src/theme/CodeBlock/WordWrapButton/styles.module.css

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+.wordWrapButton {
+  right: calc(
+    var(--ifm-pre-padding) + var(--docusaurus-code-button-size)
+  ) !important;
+}
+
+.wordWrapButtonIcon {
+  width: 1.2rem;
```

Comment:
maybe we want to use the same size for all buttongroup icons? and introduce anv env variable to customize?

Reply:
It doesn't work that way, the proportions of both icons are different, so if we use the same sizing for them, the wordWrap button will look smaller than the copyButton icon:  

![image](https://user-images.githubusercontent.com/4408379/164677099-5685665d-ef22-46ee-b5b1-24351f318b13.png)

Reply:
we can keep it this way for now, but I guess the wordWrap svg could also be cropped somehow to take full space